openslides_backend/action/actions/meeting/clone.py

```diff
@@ -79,6 +82,21 @@ def check_permissions(self, instance: dict[str, Any]) -> None:
         MeetingPermissionMixin.check_permissions(self, instance)
 
     def update_instance(self, instance: dict[str, Any]) -> dict[str, Any]:
+        organization = self.datastore.get(
+            ONE_ORGANIZATION_FQID, ["require_duplicate_from", "template_meeting_ids"]
+        )
+        if organization.get("require_duplicate_from"):
+            if not has_organization_management_level(
+                self.datastore,
+                self.user_id,
+                OrganizationManagementLevel.CAN_MANAGE_ORGANIZATION,
+            ) and not instance["meeting_id"] in (
+                organization.get("template_meeting_ids") or []
+            ):
```

Comment:
Either combine the if-conditions using and (i.e. `organization.get("require_duplicate_from") and not has_organization_management_level(self.datastore, self.user_id, OrganizationManagementLevel.CAN_MANAGE_ORGANIZATION) and not instance["meeting_id"] in (organization.get("template_meeting_ids") or [])`) or do something like the following
```suggestion
        if not instance["meeting_id"] in (
            organization.get("template_meeting_ids") or []
        ):
            organization = self.datastore.get(
                ONE_ORGANIZATION_FQID, ["require_duplicate_from", "template_meeting_ids"], lock_result=False
            )
            if organization.get("require_duplicate_from") and not has_organization_management_level(
                self.datastore,
                self.user_id,
                OrganizationManagementLevel.CAN_MANAGE_ORGANIZATION,
            ):
```

Reply:
Use 'and' to combine the if-conditionals.